sys/tools/syscalls/bsdio.lua

```diff
@@ -0,0 +1,144 @@
+--
+-- SPDX-License-Identifier: BSD-2-Clause
+--
+-- Copyright (c) 2024 Tyler Baxter <agge@FreeBSD.org>
+--
+--
+-- Thanks to Kyle Evans for his makesyscall.lua in FreeBSD which served as
+-- inspiration for this, and as a source of code at times.
+--
+-- SPDX-License-Identifier: BSD-2-Clause-FreeBSD
+--
+-- Copyright (c) 2019 Kyle Evans <kevans@FreeBSD.org>
+--
+
+util = require("util")
+
+local bsdio = {}
+
+bsdio.__index = bsdio
+
+-- Wrapper for lua write() best practice. For a simpler write call.
+function bsdio:write(line)
+	assert(self.bsdio:write(line))
+end
+
+--
+-- A write macro for the PAD64 preprocessor directive. 
+-- PARAM: bool, TRUE to pad
+-- USAGE: Pass config.abiChanges("pair_64bit") and padding will be done if 
+-- necessary. Useful for 32-bit configurations.
+--
+function bsdio:pad64(bool)
+    if bool then
+	    self:write(string.format([[
```

Comment:
I don't think you're getting anything out of `string.format()` here -- let's drop that and just use the multiline string directly as an argument to write()

Reply:
Yes, I found other instances of this. Moving forward, please let me know if you see anything else.